doc/modules/dmarc.md

```diff
@@ -28,19 +30,19 @@ Symbols added by the module are as follows:
 - `DMARC_POLICY_QUARANTINE`: Authentication failed- quarantine suggested by DMARC policy
 - `DMARC_POLICY_SOFTFAIL`: Authentication failed- no action suggested by DMARC policy
 
-Rspamd is able to store records in `redis` which could be used to generate DMARC aggregate reports but there is as of yet no available tool to generate such reports from these. Format of the records stored in `redis` is as follows:
+Rspamd has the ability to store records in `redis`, which can be used to produce DMARC aggregate reports, but there are currently no tools available to generate such reports from these records. The format of the records stored in `redis` is as follows:
 
     unixtime,ip,spf_result,dkim_result,dmarc_disposition
 
-where spf and dkim results are `true` or `false` indicating whether an aligned spf/dkim identifier was found and dmarc_disposition is one of `none`/`quarantine`/`reject` indicating policy applied to the message.
+In this format, the `spf_result` and `dkim_result` fields indicate whether an aligned SPF/DKIM identifier was found and are represented by `true` or `false`. The `dmarc_disposition` field shows the policy applied to the message and can have one of three values: `none`, `quarantine`, or `reject`.
 
-These records are added to a list named $prefix$domain where $domain is the domain which defined policy for the message being reported on and $prefix is the value of the `key_prefix` setting (or "dmarc_" if this isn't set).
+Records are inserted into a list named `$prefix$domain`, where `$domain` corresponds to the domain that has defined the policy for the reported message, and `$prefix` is determined by the `key_prefix` setting (or defaults to `dmarc_ `if `key_prefix` is not set).
 
-Keys are inserted to redis servers when a server is selected by hash value from sender's domain.
+When a hash value is assigned to a sender's domain, the corresponding key is added to the Redis server.
 
-To enable storing of report information, `reporting` must be set to `true`. Please see the section on reporting in this document for more information.
+To enable the storage of report information, the `reporting` setting must be set to `true`. For more details, please refer to the reporting section in the documentation.
 
-Actions can be forced for messages based on DMARC disposition as demonstrated in example config below.
+The example configuration below illustrates how actions can be enforced for messages based on their DMARC disposition.
```

Comment:
There should be a link to the FAQ section about the side effects of the passthrough actions. In general, such a configuration should be discouraged.